### PR TITLE
Implement version option

### LINKED ISSUE #7: geekhours --version command
Ref: https://github.com/PyCQA/pylint/blob/master/pylint/__pkginfo__.py

1. Create PyPI based packages information in `__pkginfo__.py`.
2. Load `__pkginfo.py` like https://github.com/PyCQA/pylint/blob/master/setup.py#L48 in setup.py. then run setup().
3. To create `--version`, import `__pkginfo__.py` in the command, and show its version.

### PR DESCRIPTION
**Pull request by cat-sniffer**
Implement `geekhours --version` command.
This PR closes #7.

### CODE REVIEW

**Comment by mitsutaka** [comment]
Can you let make setup installs wheel package?
```
 setup:
-       pip3 install -U pylint pytest pycodestyle yapf --user
+       pip3 install -U pylint pytest pycodestyle yapf wheel --user
```

**Comment by mitsutaka** [comment]
Fix this error when no argument specified.
```
$ geekhours
Traceback (most recent call last):
  File "/home/ubuntu/.local/bin/geekhours", line 316, in <module>
    main()
  File "/home/ubuntu/.local/bin/geekhours", line 307, in main
    args = get_args(today)
  File "/home/ubuntu/.local/bin/geekhours", line 300, in get_args
    args = parser.parse_args()
  File "/usr/lib/python3.6/argparse.py", line 1743, in parse_args
    args, argv = self.parse_known_args(args, namespace)
  File "/usr/lib/python3.6/argparse.py", line 1775, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)
  File "/usr/lib/python3.6/argparse.py", line 2010, in _parse_known_args
    ', '.join(required_actions))
TypeError: sequence item 0: expected str instance, NoneType found
```

**Review by mitsutaka — CHANGES_REQUESTED** [review]
LGTM. but pls fix what I commented above.

**Comment by YukieK** [comment]
Reflected the comments.

File: bin/geekhours [hunk 1]
```diff
@@ -182,27 +185,37 @@ def _output_in_JSON(contents: Dict) -> str:
...
+  geekhours done rm date course_name
+  geekhours sum
+  geekhours sum course
+  geekhours sum week [-c|--course [course_name]]
+  geekhours sum month [-c|--course [course_name]] """)
+
+
 def get_args(today: str):
-    """ Parsers for 'corse' and 'done' sub-commands """
+    """ Parsers for 'corse', 'done' and 'sum' sub-commands """
```
Comment (mitsutaka) [inline bin/geekhours]: s/corse/course/

Comment (YukieK) [reply bin/geekhours]: Fixed.

**Review by mitsutaka — APPROVED** [review]
LGTM

**Pull request merged**